components/input/src/auro-input.js

```diff
@@ -0,0 +1,213 @@
+            minlength="${ifDefined(this.minLength ? this.minLength : undefined)}"
+            inputMode="${ifDefined(this.inputMode ? this.inputMode : undefined)}"
+            ?required="${this.required}"
+            ?disabled="${this.disabled}"
+            aria-describedby="${this.uniqueId}"
+            aria-invalid="${this.validity !== 'valid'}"
+            placeholder=${this.getPlaceholder()}
+            lang="${ifDefined(this.lang)}"
+            ?activeLabel="${this.activeLabel}"
+            spellcheck="${ifDefined(this.spellcheck ? this.spellcheck : undefined)}"
+            autocorrect="${ifDefined(this.autocorrect ? this.autocorrect : undefined)}"
+            autocapitalize="${ifDefined(this.autocapitalize ? this.autocapitalize : undefined)}"
+            autocomplete="${ifDefined(this.autocomplete ? this.autocomplete : undefined)}"
+            part="input"
+            />
+        </div>
+        <div
+          class="notificationIcons"
+          part="notificationIcons"
+          ?hasValue="${this.hasValue}">
+          ${this.validity && this.validity !== 'valid' ? html`
+            <div class="notification alertNotification">
+              <${this.iconTag}
+                category="alert"
+                name="error-stroke"
+                customSize
+                error>
+              </${this.iconTag}>
+            </div>
+          ` : undefined}
+          ${this.hasValue ? html`
+            ${this.type === 'password' ? html`
+              <div class="notification">
+                <${this.buttonTag}
+                  variant="flat"
+                  aria-hidden="true"
+                  tabindex="-1"
+                  @click="${this.handleClickShowPassword}"
+                  class="notificationBtn passwordBtn">
+                  <${this.iconTag}
+                    category="interface"
+                    name="hide-password-stroke"
+                    customSize
+                    customColor
+                    ?hidden=${!this.showPassword}>
+                  </${this.iconTag}>
+                  <${this.iconTag}
+                    category="interface"
+                    name="view-password-stroke"
+                    customSize
+                    customColor
+                    ?hidden=${this.showPassword}>
+                  </${this.iconTag}>
+                </${this.buttonTag}>
+              </div>
+            ` : undefined}
+            ${!this.disabled && !this.readonly ? html`
+              <div class="notification">
+                <${this.buttonTag}
+                  variant="flat"
+                  class="notificationBtn clearBtn"
+                  aria-hidden="true"
+                  tabindex="-1"
+                  @click="${this.handleClickClear}">
+                  <${this.iconTag}
+                    customColor
+                    category="interface"
+                    name="x-lg"
+                    customSize>
```

Comment:
Remove customSize attribute when version of `auro-icon` is updated.

Reply:
Updates to `auro-input` are out of scope for this PR since that code will need to be copied-and-pasted again once the new `form` component launches, and we deprecated the older `auro-input` component.

I will create separate tickets to track these changes, however.